### PR TITLE
fix: fail to open log file even if user is in haclient group (bsc#1204670)

### PR DESCRIPTION
**Pull request by nicholasyang2022**
The file had been created with umask 0022 in usual so that it was not group-writable.

Call chown and chmod explicitly to fix it.

### CODE REVIEW

**Comment by nicholasyang2022** [comment]
See also: https://build.opensuse.org/request/show/1031002 for packaging changes to create log directory.

File: crmsh/log.py [hunk 1]
```diff
@@ -432,10 +424,17 @@ def setup_logging(only_help=False):
     if only_help:
         LOGGING_CFG["handlers"]["file"] = {'class': 'logging.NullHandler'}
     else:
-        setup_directory_for_logfile()
+        # dirname(CRMSH_LOG_FILE) should be created by package manager during installation
```
Comment (fmherschel) [inline crmsh/log.py:427]: Looks good for me.

**Comment by nicholasyang2022** [comment]
Cherry pick a commit from #991, which fixes ownership and mode for new files created in log rotation.

**Review by liangxin1300 — APPROVED** [review]
Great work!
Thank you @nicholasyang2022

**Pull request merged**